clickhouse/tests/test_clickhouse.py

```diff
@@ -1,7 +1,9 @@
 # (C) Datadog, Inc. 2019-present
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
+import mock
```

Comment:
is this python2 or python3? python3 added mock into the standard library as `unittest.mock`. if other tests in this repo use `mock` with `import mock`, then i'd keep it as is